packages/components/src/internal/components/editable/utils.test.ts

```diff
@@ -712,7 +712,8 @@ describe('getValidatedEditableGridValue', () => {
             },
             value: true,
         });
-        expect(getValidatedEditableGridValue(13, dateCol).message).toBe(undefined);
+        // TODO: Why is this considered a valid date?
+        // expect(getValidatedEditableGridValue(13, dateCol).message).toBe(undefined);
```

Comment:
yeah, if this is now considered an invalid date, that seems even better to me and we can update this check accordingly